doc/manuals/user/context_providers.md

```diff
@@ -70,17 +69,49 @@ curl localhost:1026/v2/entities/Street4/attrs/temperature?type=Street -s -S \
     -H 'Accept: application/json' -d @- | python -mjson.tool
 ``` 
 
-* Orion doesn't know the Street 4 temperature, but it knows (due to
+* Orion doesn't know the Street4 temperature, but it knows (due to
   the registration in the previous step) that the Context Provider at
-  <http://sensor48.mycity.com/v1> does know about the Street 4 temperature, so it forwards the query
+  <http://sensor48.mycity.com/v1> does know about the Street4 temperature, so it forwards the query
   (message number 3) to the URL
-  <http://sensor48.mycity.com/v1/queryContext> (i.e. the URL used in
-  the Providing Application field at registration time, plus the
-  "/queryContext" operation). Note that the query is forwarded using
-  NGSIv1 format although the original request from the client used NGSIv2
-  (NGSIv1 is deprecated, but we need to do so as there is yet lack of support for NGSIv2 based forwarding, see
-  [this issue about it](https://github.com/telefonicaid/fiware-orion/issues/3068)).
+  <http://sensor48.mycity.com/v1/queryContext> (i.e., the URL used in
+  the Providing Application field at registration time, and adding
+  "/queryContext" to the URL PATH).
+  The query is forwarded using NGSIv2 format by default.
+  It doesn't matter whether NGSIv1 or NGSIv2 was used in the query, what decides the format of the
+  forward is what was demanded by the registration.
+  To have the NGSIv1 format used for the forwarding, the field "legacyForwarding" must be set to `true` in the registration:
+  ```
+  "legacyForwarding": true
```

Comment:
context_providers.md should provide a description based in NGSIv2, as if NGSIv1 didn't exist. The legacyForwarding explanation is provided in "implementation notes".

Reply:
That we should do once NGSIv2 is fully implemented.
For now, we only have queries in V2.
So, postponed?

Reply:
The example in this .md is covering only queries so it could be done in this PR and, at the same time, minimize the times we touch the documentation (which makes also life easier for people that does translations on the documentation).

Reply:
Attempt made in 5404560b1